### ISSUE TITLE
[Documentation] Analytics Data Dictionary on Research Folder

### ISSUE THREAD

**Issue by joanneesteban** [department-of-veterans-affairs/va.gov-team#849]
## User Story
As a product owner, I want to understand how metrics are implemented and defined, so I can self-service my data requests within Google Analytics. 

## Goal
Create first, barebones data dictionary in Github.

## Acceptance Criteria
- [ ] First draft of transaction metrics uploaded as Markdown

## Definition of Done
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by nedierecel** [comment]
@jonwehausen @joanneesteban This is a draft of the global and product-specific events and custom dimensions/metrics that we will spread into a few sprints: 

https://docs.google.com/spreadsheets/d/13cSftwf1yAvd52qqzsS_G8tQTCDaTcSL42AwVQMX9NY/edit?usp=drive_web&ouid=110723540932834422390

The final format will be in markdown (with stretch of getting screenshots also) and live in the Analytics folder.

**Comment by nedierecel** [comment]
@joanneesteban Should we rescope this task to define how we want to structure our new GitHub documentation? 

Here is our current structure: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/Platform/Teams/Analytics%20and%20Insights

**Comment by joanneesteban** [comment]
@nedierecel Yeah, I think that's a separate task (since our Github documentation will need to be completely restructured too). We'll keep this as adding the first draft of the data dictionary to our documentation.

**Comment by nedierecel** [comment]
@joanneesteban I tried to create a `google-analytics` folder within `platform` but blocked on access. Can you help with permissions so I can commit changes?

![image.png](https://images.zenhubusercontent.com/5b295b9ff378ea1890b3d77e/4591c163-2406-428b-a4cc-7739aaefa23a)

**Comment by nedierecel** [comment]
First draft uploaded for `Defining Success` resources under Platform>Analytics>Google Analytics - https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/google-analytics/defining-success.md

More to come!

**Comment by nedierecel** [comment]
@joanneesteban @jonwehausen I was able to take a first stab at the Analytics Review checklist: https://docs.google.com/document/d/1anZR9VLbJcNfP2nqjiIoFW4lbx7j3_w8oXxW9zTwAZo/edit

There are still gaps on turnaround times but wanted to get something for us to review.

**Comment by nedierecel** [comment]
@joanneesteban New resource available in our platform folder:

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/google-analytics/resources-navigating-google-analytics.md

**Comment by nedierecel** [comment]
@joanneesteban 

New resource available:
- [Rules of Engagement - Request Review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/rules-of-engagement-request-review.md)

**Comment by joanneesteban** [comment]
@joanneesteban help with readme.md for top level platform analytics folder.a

**Comment by nedierecel** [comment]
@joanneesteban We can wrap further documentation updates into an Epic, however, please see the latest documentation for the Google Analytics platform folder. The readme in the `/platform/analytics/google-analytics` folder includes several .md files organized by how the VFS team should work through the documentation, from onboarding to build to analysis.

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/google-analytics/readme.md

There is also a data dictionary for multi-step forms. Other areas like navigation, authentication, etc. can be put into future sprints.
